src/pretalx/schedule/models/room.py

```diff
@@ -18,6 +20,11 @@ class Room(LogMixin, models.Model):
         to="event.Event", on_delete=models.PROTECT, related_name="rooms"
     )
     name = I18nCharField(max_length=100, verbose_name=_("Name"))
+    guid = models.UUIDField(
```

Comment:
Do we want a `unique_together` restriction here to make sure every guid appears only once per event?